Components/avatar.css

```diff
@@ -0,0 +1,20 @@
+.avatar {
+  width: 45px;
+  height: 45px;
+  border-radius: 50%;
+}
+.sm {
+  width: 30px;
+  height: 30px;
+}
+.lg {
```

Comment:
Be more specific with class names, here it is unclear whether sm, lg, m is for padding, margin, or width. `avatar-sm` would be more explanatory.

Reply:
Thanks Jantu, will correct it